languages/wp-job-manager.pot

```diff
@@ -572,15 +572,15 @@ msgid "Page numbered links"
 msgstr ""
```

Comment:
We don't update the pot file during development. It is automatically generated during release.

Reply:
Thanks for the input!